element_animal/export/nwb.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+import json
+import pynwb
+
+from element_animal import subject
+
+
+def subject_to_nwb(subject_key):
+    subject_query = subject.Subject & subject_key
+    subject_query = subject_query.join(subject.Subject.Line, left=True)
+    subject_query = subject_query.join(subject.Subject.Strain, left=True)
+    subject_query = subject_query.join(subject.Subject.Source, left=True)
+    subject_info = subject_query.fetch1()
+
+    return pynwb.file.Subject(
+        subject_id=subject_info.pop('subject'),
+        sex=subject_info.pop('sex'),
+        date_of_birth=datetime.combine(subject_info.pop('subject_birth_date'),
+                                       datetime.strptime('00:00:00', '%H:%M:%S').time()),
+        description=json.dumps(subject_info, default=str),
+        genotype=' x '.join((subject.Line.Allele * subject.Subject.Line & subject_key).fetch('allele')),
+        species='Mus musculus'
```

Comment:
Is it safe to hardcode pieces in the export function? My instinct is to instead set as a default within the Line or Subject class, including a note that NWB prefers latin names.

Reply:
PR to [ttngu207/element-animal](https://github.com/ttngu207/element-animal) will resolve this

Reply:
where are we on this?